include/exadg/incompressible_navier_stokes/postprocessor/line_plot_calculation.cpp

```diff
@@ -64,45 +89,43 @@ LinePlotCalculator<dim, Number>::evaluate(VectorType const & velocity,
     // precision
     unsigned int const precision = data.line_data.precision;
 
-    // loop over all lines
-    for(typename std::vector<std::shared_ptr<Line<dim>>>::const_iterator line =
-          data.line_data.lines.begin();
-        line != data.line_data.lines.end();
-        ++line)
+    LinearAlgebra::distributed::Vector<double> ghosted_pressure(pressure.get_partitioner());
```

Comment:
The class ``LinePlotCalculator`` has the template parameter ``Number``, so why do you use ``double`` here?

Reply:
This is an old bug in deal.II, which prevented evaluation with `float` numbers. When I update the code, I will put this into an `ifdef` for the 9.3 version of deal.II.